tough/src/schema/key.rs

```diff
@@ -33,7 +33,7 @@ use std::str::FromStr;
 ///  * `Rsa`: PUBLIC is in PEM format and a string. All RSA keys must be at least 2048 bits.
 ///  * `Ed25519`: PUBLIC is a 64-byte hex encoded string.
 ///  * `Ecdsa`: PUBLIC is in PEM format and a string.
-#[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
+#[derive(Debug, Clone, Deserialize, Serialize, Eq, PartialEq)]
```

Comment:
I'm assuming this was a clippy thing?

Reply:
Yeah, they seem to have added a lint to suggest you derive `Eq` in some cases where you are already deriving `PartialEq`